apps/wallet-mobile/src/TxHistory/PairedBalance.tsx

```diff
@@ -14,8 +14,9 @@ import {Quantities} from '../yoroi-wallets/utils'
 type Props = {
   amount: Balance.Amount
   isPrivacyOff?: boolean
+  style?: TextStyle
 }
-export const PairedBalance = React.forwardRef<ResetErrorRef, Props>(({isPrivacyOff, amount}, ref) => {
+export const PairedBalance = React.forwardRef<ResetErrorRef, Props>(({isPrivacyOff, amount, style}, ref) => {
```

Comment:
maybe to don't pass the same style to different components, or even many styles errorStyle, amountStyle, create a `variant`?

Reply:
They me be different components, but they're all Text in the end. And all of them sharing the same style already. What's the benefit of variants over style? I would have to make the variants here instead of on usage.

Reply:
But they are not different components, they are all the same one. It's just that the component is over-complicated with sub components when all it does is either render null or Text, and change the value inside the Text. But the styles of the Text are always the same, and are the ones I want to override from outside.
If I created a variant it would have to be applied to all the sub components and check on each what variant it is.